src/components/elements/carousel/carousel.js

```diff
@@ -95,11 +109,21 @@ export const Carousel = ({
         stop()
     }, [embla, stop])
 
+    const scrollTo = useCallback(
+        (index) => {
+            if (!embla) return
+            embla.scrollTo(index)
+            stop()
+        },
+        [embla, stop],
+    )
+
```

Comment:
The alignment of this callback is weird 😅 Can you run `npm run format` for that

Reply:
Sure ~ will run